docs/source/operations/projections/somerc.rst

```diff
@@ -28,3 +28,7 @@ Parameters
 .. include:: ../options/x_0.rst
 
 .. include:: ../options/y_0.rst
+
+.. option:: +latc=<value>
+
+   Normal latitude of the ellipsoid
```

Comment:
A bit more explanation would be needed, like an extract from the summary of the paper + a link to it.